api-client/src/test/java/com/xing/api/internal/json/SafeCalendarJsonAdapterTest.java

```diff
@@ -335,9 +341,70 @@ private TimeZone parseTimeZone(String dateString) throws IOException {
         return calendar.getTimeZone();
     }
 
+    @Test
+    public void allThreadMustHaveTheirOwnThreadLocalSimpleDateFormat() {
+        int numberOfParties = 10;
+        final CyclicBarrier barrier = new CyclicBarrier(numberOfParties + 1);
+        final List<SimpleDateFormat> list = Collections.synchronizedList(new ArrayList<SimpleDateFormat>());
+        class Worker implements Runnable {
```

Comment:
Uh, didn't knew you can declare a class in a method - learn a new thing every day! :)